Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Prisma_Cloud_Enrichment.yml

```diff
@@ -555,175 +555,8 @@ tasks:
     view: |-
       {
         "position": {
-          "x": 710,
```

Comment:
Need to make changes from https://github.com/demisto/content/pull/28908 are reflected here

Reply:
resolved